### PR TITLE
Move pushblock with input system integration into the Project.

### PR DESCRIPTION
**Pull request by surfnerd**
### Proposed change(s)

- Remove input sample project and move it into 'Project'

### Types of change(s)

- [x] Code refactor
- [x] Other (please describe) - see above.

### CODE REVIEW

File: Project/Packages/manifest.json [hunk 1]
```diff
@@ -1,18 +1,23 @@
 {
   "dependencies": {
+    "com.unity.ide.rider": "3.0.5",
+    "com.unity.inputsystem": "1.1.0-preview.3",
     "com.unity.ml-agents": "file:../../com.unity.ml-agents",
     "com.unity.ml-agents.extensions": "file:../../com.unity.ml-agents.extensions",
     "com.unity.nuget.newtonsoft-json": "2.0.0",
     "com.unity.test-framework": "1.1.22",
+    "com.unity.toolchain.macos-x86_64-linux-x86_64": "0.1.20-preview",
```
Comment (chriselion) [inline Project/Packages/manifest.json:9]: Should this (and the rider package) be here?

**Comment by chriselion** [comment]
Is there enough duplicated code that this could changes to be a new scene in PushBlock? Although maybe the original PushBlock is going away soon (to be combined with WallJump) - @vincentpierre any thoughts?

**Comment by surfnerd** [comment]
The two agents are very different since I removed the code that controls the agent out of the original push block.  I did cringe at the duplication of GoalDetect.

File: Project/Packages/manifest.json (hunk 1 above)
Comment (surfnerd) [reply Project/Packages/manifest.json:9]: this was added automatically.  I added rider back because it makes quite a difference in development experience.

**Review by chriselion — APPROVED** [review]

**Pull request merged**